### PR TITLE
Replace accidental U+FFFC with space in GH Action lock.yml

### PR DESCRIPTION
**Pull request by ejona86**

### CODE REVIEW

File: .github/workflows/lock.yml [hunk 1]
```diff
@@ -17,4 +17,4 @@ jobs:
         with:
           github-token: ${{ github.token }}
           issue-lock-inactive-days: 90
-￼         pr-lock-inactive-days: 90
+          pr-lock-inactive-days: 90
```
Comment (voidzcy) [inline .github/workflows/lock.yml:20]: What's happening here? I literally don't see anything different lol. Is that non-printable unicode?

**Review by dapengzhang0 — APPROVED** [review]

**Comment by dapengzhang0** [comment]
How did you notice it? Does the original one not work?

**Comment by ejona86** [comment]
Yeah, the original failed complaining about the first character of the line. https://github.com/grpc/grpc-java/actions/runs/900981749 Searching for " " (space) in my text editor I saw it didn't highlight that first character. Looking at the file with `od` showed it was a U+FFFC (replacement character) in UTF-8 encoding.

**Comment by BloggerMargaret** [comment]
Good day! I hope you all and your families are safe? The Covid-19 situation is really getting worst all over the globe, we all pray to almighty god to strength us, so that we can stay alive to testify, I brought good news that everyone can benefit from and also help the privilege, if you are a good kindest person, that also want to make the world a better place to live, now the freemasonry society are selecting the kindest people that are interested to join the society, and the good part is that freemasonry society they don't involved their self into evil things, they are not like the illuminati society, freemasonry society believe in god, they are grouped of people that come together to evaluate people and others original condition, once you become a member of the society they have sitting amount of benefit that we be given to you as the new member, and you should use 10% of the benefit to assist the privilege, and summit the credential to the lordship, I urge you if you have interest, find the official contact address below, you can be one of the lucky random, everyone has inside them a piece of good news, the good news is you don’t know how great you can be! How much you can love! What you can accomplish! And what your potential is, stay strong, stand up, have a voice.

WhatsApp; +31 687329133

Mail Address; info.masonic.oregon@gmail.com

Website Address; https://masonicoregon.wordpress.com

**Pull request merged**